py3-prometheus-client.yaml

```diff
@@ -17,7 +27,7 @@ environment:
       - busybox
       - ca-certificates-bundle
       - py3-setuptools
```

Comment:
Believe this should be `py3-supported-setuptools`

Reply:
Ah, yes. I missed it. I think the py/pip-build-install pipeline actually provides it, that's probably why the build went through. Good catch!